README.md

```diff
@@ -3,17 +3,28 @@
 
 [![Build Status](https://circleci.com/gh/iodide-project/pyodide.png)](https://circleci.com/gh/iodide-project/pyodide)
 
-The Python scientific stack, compiled to WebAssembly.
+Pyodide is cPython compiled to WebAssembly and running in the browser.  It can run any pure Python package, and 
+includes over 37 of Python's [best Data Science libraries](./packages) compiled to WebAssembly.
 
-It provides transparent conversion of objects between Javascript and Python.
-When inside a browser, this means Python has full access to the Web APIs.
+Pyodide works great with Javascript. Proxy objects make it easy for Python and Javascript to interoperate. 
```

Comment:
> Proxy objects make it easy for Python and Javascript to interoperate. 

It's really the "transparent conversion" that makes it easy to interoperate, of which Proxies are just one part.  I think we just leave the sentence about Proxies out here -- it's kind of an implementation detail.